### PR TITLE
Feature/flow 1182 update documentation

### PR DESCRIPTION
**Pull request by rwe17**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Swagger-ui
+
+## Swagger integration
+
+* import `swagger-asset` and `leanix-swagger-ui` library into your microservice
```
Comment (fabianboeller) [inline README.md]: Do you mean `dropwizard-assets` instead of `swagger-asset`?

File: pom.xml [hunk 2]
```diff
@@ -1,20 +1,49 @@
...
         <groupId>net.leanix.poms</groupId>
         <artifactId>common-parent</artifactId>
-        <version>0.8.2</version>
+        <version>2.0.1</version>
         <relativePath />
     </parent>
+
+    <properties>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
```
Comment (fabianboeller) [inline pom.xml]: Should this be 1.11?

File: README.md (hunk 1 above)
Comment (rwe17) [reply README.md]: Thanks, this is dropwizard-assets.

File: pom.xml (hunk 2 above)
Comment (rwe17) [reply pom.xml]: We not not use functionality of java 11 and therefor i would suggest to stay on java 8.

**Pull request merged**